src/Database/Connection.php

```diff
@@ -127,6 +128,9 @@ public function __construct($config)
      */
     public function __destruct()
     {
+        if ($this->_transactionStarted) {
```

Comment:
This need to check that the Log class actually exists, as it is possible to use the Database package without Log